### DISCUSSION TITLE
How can I forward a request to another service

### DISCUSSION THREAD

**Discussion by keJ2011** [Kong/kong#9945] [Help] [answered]
hi, I have two service: service1、service2 , default request will go throw the service1, I have a traffic plugin, when request meets some conditions, i hope it will be forward to service2.   because i hope the request will hand by the service2's plugin, so i can't use the kong.service.set_upstream(host) function.

**Comment by StarlightIbuki** [comment, marked as answer]
You could try to proxy the request to the gateway itself, with the URL which will hit your desired service(2).

**Reply by keJ2011** [reply]
thank you, it salve my problem.
